priv/repo/migrations/20161024204753_remove_package_downloads_pk.exs

```diff
@@ -0,0 +1,84 @@
+defmodule HexWeb.Repo.Migrations.RemovePackageDownloadsPk do
+  use Ecto.Migration
+
+  def up do
```

Comment:
@ericmj could you double-check this change especially? I basically combined these 2 migrations:
- https://github.com/hexpm/hex_web/blob/master/priv/repo/migrations/20160317073758_add_package_downloads_pk.exs
- https://github.com/hexpm/hex_web/blob/master/priv/repo/migrations/20160530111051_optimize_downloads_view_indicies.exs